sdks/java/io/jms/src/main/java/org/apache/beam/sdk/io/jms/JmsCheckpointMark.java

```diff
@@ -34,7 +37,7 @@
 public class JmsCheckpointMark implements UnboundedSource.CheckpointMark {
```

Comment:
Do take a quick pass over this class and try to restrict the visible APIs as much as possible.

In general, anything that's an internal implementation detail should be `private` or at least package-visible (if needed for testing). This will make our lives much easier once we start having a backwards-compatible guarantee.